airbyte-cdk/python/airbyte_cdk/sources/declarative/requesters/http_requester.py

```diff
@@ -547,6 +548,8 @@ def _try_get_error(value: Any) -> Any:
                     or value.get("errors")
                     or value.get("failures")
                     or value.get("failure")
+                    or value.get("details")
```

Comment:
it would be a good idea to share this code with HttpStream by moving the logic to a function that can be shared. can you do that as a follow up PR?

Reply:
It's good idea, will make this as a next task